gazebo_msgs/srv/GetModelState.srv

```diff
@@ -4,6 +4,10 @@ string relative_entity_name          # return pose and twist relative to this en
                                      # be sure to use gazebo scoped naming notation (e.g. [model_name::body_name])
                                      # leave empty or "world" will use inertial world frame
 ---
+Header header                        # Standard metadata for higher-level stamped data types.
+                                     # * header.seq holds the number of requests since the plugin started
```

Comment:
The seq value should not be used. If this is an important value you should actually have a field for it. 

Since you're not using the frame_id, you should consider just putting in a named time field.